libs/SFMLComponents/GraphicsRectangleComponent.hpp

```diff
@@ -0,0 +1,31 @@
+/*
+** EPITECH PROJECT, 2022
+** rtype
+** File description:
+** GraphicsRectangleComponent
+*/
+
+#ifndef GRAPHICSRECTANGLECOMPONENT_HPP_
+#define GRAPHICSRECTANGLECOMPONENT_HPP_
+
+#include "Component/Component.hpp"
+#include <SFML/Graphics.hpp>
+
+namespace ecs
+{
+    /// @brief This component class stores a Graphical SFML rectangle shape.
+    /// This class is created in order to draw a rectangle shape on a screen.
+    class GraphicsRectangleComponent : public Component {
+        public:
+            /// @brief The Graphical SFML rectangle shape to be rendered.
+            sf::RectangleShape shape;
```

Comment:
How do you create it? With which parameters?

Reply:
Directly by using the shape

Reply:
Can you not specify its size, color, etc... to its creation? 
If possible, you should add the possibility to do so (Its size at least).